acct-user/logcheck/logcheck-0-r2.ebuild

```
@@ -6,8 +6,16 @@ EAPI=7
 inherit acct-user
 
 DESCRIPTION="User for app-admin/logcheck"
+IUSE="systemd"
 
 ACCT_USER_GROUPS=( "logcheck" )
 ACCT_USER_ID="284"
 
 acct-user_add_deps
+
+pkg_setup() {
+	# required to allow the user "logcheck" to run journalctl
```

```suggestion
	# Allow the 'logcheck' user to view the systemd journal.
```

Everyone can run `journalctl`, the question is if they are able to view it ;)